src/app.js

```diff
@@ -5,25 +5,26 @@ const app = express();
 const hbars = require('express-handlebars');
 
 const path = require('path');
+const favicon = require('serve-favicon');
 const routes = require('./controllers');
 const bodyparser = require('body-parser');
 
-
-app.use(bodyparser.json());
-app.use(bodyparser.urlencoded({ extended: false }));
-
-app.use(express.static(path.join(__dirname, '..', 'public')));
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'hbs');
-app.engine('hbs', hbars({
-  extname: 'hbs',
-  layoutsDir: path.join(__dirname, 'views', 'layouts'),
-  partialsDir: path.join(__dirname, 'views', 'partials'),
-  defaultLayout: 'main',
-}));
-
-app.set('port', process.env.PORT || 4000);
-app.use(routes);
-
-
-module.exports = app;
+module.exports = app
+  .use(bodyparser.json())
+  .use(bodyparser.urlencoded({
+    extended: false,
+  }))
+
+  .use(express.static(path.join(__dirname, '..', 'public')))
+  .set('views', path.join(__dirname, 'views'))
+  .set('view engine', 'hbs')
+  .engine('hbs', hbars({
+    extname: 'hbs',
+    layoutsDir: path.join(__dirname, 'views', 'layouts'),
+    partialsDir: path.join(__dirname, 'views', 'partials'),
+    defaultLayout: 'main',
+  }))
+
+  .set('port', process.env.PORT || 8080)
```

Comment:
return the port to 4000 before you push your task

Reply:
ok